client/app/components/Header/HeaderProfile.scss

```diff
@@ -0,0 +1,45 @@
+
+  .header {
```

Comment:
This is still a good refactor! You'll want to import `@import "../../styles/_global.scss";` at the top of this file. 

I think you just want to keep what's at line 10 and beyond.